src/popup/components/TransactionOverview.vue

```diff
@@ -186,10 +188,10 @@ export default defineComponent({
 
       const sdk = await getSdk();
       const { bytecode } = await sdk.getContractByteCode(innerTx.value.contractId);
-      const txParams: ITx = await sdk.compilerApi.decodeCalldataBytecode({
-        bytecode,
-        calldata,
-      });
+      const txParams: ITx = await postJson(
+        `${activeNetwork.value.compilerUrl}/decode-calldata/bytecode`,
+        { body: { bytecode, calldata } },
+      );
```

Comment:
Nice move. Add a todo to replace this with SDK method whenever we update the SDK version that supports this method.